packages/mattermost-redux/src/actions/admin.ts

```diff
@@ -445,6 +445,24 @@ export function removeLicense(): ActionFunc {
     });
 }
 
+export function getPrevTrialLicense(): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        let data;
+        try {
+            data = await Client4.getPrevTrialLicense();
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            return {error};
+        }
+
+        dispatch(batchActions([
+            {type: AdminTypes.PREV_TRIAL_LICENSE_SUCCESS, data},
```

Comment:
no need for batch actions since it's only one action we are dispatching

Reply:
Nice catch! FIxed.